Server/mods/deathmatch/logic/packets/CChatEchoPacket.h

```diff
@@ -48,3 +48,16 @@ class CChatEchoPacket : public CPacket
     SString       m_strMessage;
     bool          m_bColorCoded;
 };
+
+class CChatClearPacket : public CPacket
+{
+public:
+    CChatClearPacket() {};
+
+    // Chat uses low priority channel to avoid getting queued behind large map downloads #6877
+    ePacketID               GetPacketID(void) const { return PACKET_ID_CHAT_CLEAR; };
+    unsigned long           GetFlags(void) const { return PACKET_LOW_PRIORITY | PACKET_RELIABLE | PACKET_SEQUENCED; };
+    virtual ePacketOrdering GetPacketOrdering(void) const { return PACKET_ORDERING_CHAT; }
+
+    bool Write(NetBitStreamInterface& BitStream) const { return true; };

```

Comment:
Are you using the `Write` function? It's not a pure virtual function, can you remove it?